src/x/instrument/string_list_emitter.go

```diff
@@ -40,101 +40,107 @@ var (
 	errStringListEmitterNotStarted     = errors.New("string list emitter: not running")
 )
 
-// StringListEmitter emits a gauge where its tags indicate the order of a
-// list of strings.
+// StringListEmitter emits a gauges indicating the order of a list of strings.
 type StringListEmitter struct {
 	sync.Mutex
 	running   bool
 	doneCh    chan bool
 	scope     tally.Scope
-	gauge     tally.Gauge
+	gauges    []tally.Gauge
 	name      string
 	tagPrefix string
 }
 
-// NewStringListEmitter returns a StringListEmitter. The name and tagPrefix
-// arguments are used by the Start() and UpdateStringList() function to set
-// the name and tags on the gauge.
-func NewStringListEmitter(scope tally.Scope, name, tagPrefix string) *StringListEmitter {
-	gauge := tally.NoopScope.Gauge("blackhole")
+// NewStringListEmitter returns a StringListEmitter.
+func NewStringListEmitter(scope tally.Scope, name string) *StringListEmitter {
+	gauge := []tally.Gauge{tally.NoopScope.Gauge("blackhole")}
```

Comment:
nit: Since this is a slice now that you loop over and .Inc each one this could just be a zero length list of gauges I assume?